CalculatorModel.kt

```diff
@@ -0,0 +1,112 @@
+package com.example.nicocalculator.mvp.model
+
+import com.example.nicocalculator.Key
+
+open class CalculatorModel {
+
+    private var textResult = RESULTS_DEFAULT_TEXT
+    private var textOperations = "8*9+6*3*3+5"

```

Comment:
what is this for?

Reply:
It was a testing operation, the correct value is OPERATIONS_DEFAULT_TEXT

Reply:
Done